cmd/util.go

```diff
@@ -82,7 +70,8 @@ func ps(proc string) string {
 	cmd := exec.Command("/bin/ps", "-C", proc, "-o", "cmd", "--no-headers")
 	out, err := cmd.Output()
 	if err != nil {
-		continueWithError(fmt.Errorf("%s: %s", cmd.Args, err), "")
+		glog.V(2).Info(err)
+		glog.V(2).Info(fmt.Errorf("%s: %s", cmd.Args, err))
```

Comment:
These logs will only show up if error logging is level 2 or higher; also logs might be going somewhere else like to a file rather than stdout or stderr. So I think we should stick with the original behaviour of printing the message to stderr.

```suggestion
		glog.V(2).Info(err)
		fmt.Fprintf(os.Stderr, "%s: %s", cmd.Args, err))
```